utils/model.py

```diff
@@ -63,6 +71,36 @@ def load_model(args: SakuraModelConfig):
     if args.llama_cpp:
         from llama_cpp import Llama
 
+    if args.vllm:
+        from vllm import AsyncEngineArgs, AsyncLLMEngine, LLM
+        from vllm.utils import Counter
+
+        class MixLLMEngine(LLM):
+            "an AsyncLLMEngine unwrapper for flexible generation"
+            def __init__(self, llm_engine: AsyncLLMEngine):
+                self.llm_engine = llm_engine.engine
+                self.async_engine = llm_engine
+                self.request_counter = Counter()
+
+            def stream_generate(self, prompt, sampling_params):
+                import asyncio
+                import nest_asyncio
```

Comment:
Is this package `nest_asyncio` installed as a dependency of package already in requirements.txt or a new package?

Reply:
a new package, so I just put forth a code snippet to get rid of it.